docs/application/dotnet/get-started/iot/first-app.md

```diff
@@ -0,0 +1,221 @@
+# Create Your First Tizen IoT .NET Application
+
+The Tizen .NET framework allows you to easily and efficiently create applications for Tizen. Study the following instructions to help familiarize yourself with the Tizen .NET application development process. With the instructions, you can create and run a basic .NET application, which displays some texts on the screen without any user interaction.
+
+1.  Before you get started with developing Tizen applications, set up the [development environment](../../../vstools/install.md).
+
+2. [Create a Project](#create-a-project) using Visual Studio.
+
+    This step shows how you can use a predesigned project template that creates all the basic files and folders required for your project.
+
+3. [Build Your Application](#build-your-application).
+
+    After you have implemented code for the features you want, this step shows how you can build the application to validate and compile
+    the code.
+
+4. [Deploy and Run Your Application](#deploy-and-run-your-application).
+
+    This step shows how you can deploy and run the application on the reference board.
+
+5. [Understand Your Application](#understand-your-application).
+
+    This step shows how you can enhance your application by creating a UI and making small alterations to improve the usability of the application.
+
+6. [Package Your Application](#package-your-application).
+
+    This step shows how you can package your application.
+
+## Create a Project
+
+The following example shows you how to create and configure a basic Tizen .NET application project in Visual Studio. An application project contains all the files that make up an application.
+
+The following figure illustrates the output of the application. 
+
+![Application running on the reference board](media/cs_first_building_device.png)
+
+The application screen displays a message, **Welcome to Xamarin Forms!** and there is no user interaction.
+
+To create a new Tizen .NET project:
+
+1.  Launch Visual Studio 2019.
+
+2. In the Visual Studio menu, select **File \> New \> Project**.
+
+    ![Create a new project](media/cs_first_creating_menupath.png)
+
+    A New Project window appears.
+
+3. Select **C\#** from languages and **Tizen** from platforms, select **Tizen XAML App (Xamarin.Forms)** template and click **Next**.
+
+    ![Select a template](media/cs_first_creating_new_project.png)
+
+    Configure the project properties and click **Create**. You can enter **Project name**, **Location**, **Solution**, and **Solution name**.
+
+    ![Select a template](media/cs_first_creating_configure_project.png)
```

Comment:
\Tizen\source\repos   This is windows path 

/Tizen/source/repos   This is Linux path

We want slashes instead of special characters, isn't it? 

@mijin85cho can you help me understand what you are trying to say here such that I too can learn.

Reply:
@nawab-samsung Sorry for the confusion. Yes, I want to change the ￦ mark to \\.

￦ is used in korean windows os, so the image is requested to be captured again on windows os in Eng mode.

**Example**  : https://docs.tizen.org/application/dotnet/get-started/mobile/first-app/

![image](https://user-images.githubusercontent.com/9958374/83718646-73594680-a670-11ea-9084-b19543008dfd.png)

Reply:
@mijin85cho thanks, I really appreciate that. It is a learning to know that special character￦is Korean alternative of /. 

Thanks a lot for always helping me learn these new things.

Reply:
I fixed the image.

Reply:
> Same issue to cs_first_creating_project_wizard_common.png and cs_first_building_sdb_prompt.png

Thanks for the update, @xbldx.

Also, **cs_first_creating_project_wizard_common.png** and **cs_first_building_sdb_prompt.png** have the same issue. 
These images are containing ￦, too.
Would you please update them also?

![image](https://user-images.githubusercontent.com/9958374/83738500-5718d200-a68f-11ea-98e8-7c6b6d099c02.png)


![image](https://user-images.githubusercontent.com/9958374/83738417-42d4d500-a68f-11ea-91dd-b67c1afdf9d5.png)